test/parallel/test-zlib-random-byte-pipes.js

```diff
@@ -150,5 +150,5 @@ const gunz = zlib.createGunzip();
 inp.pipe(gzip).pipe(gunz).pipe(out);
 
 out.on('data', common.mustCall((c) => {
-  assert.strictEqual(c, inp._hash, 'hashes should match');
+  assert.strictEqual(c, inp._hash, `Hash '${c}' equals '${inp._hash}'.`);
```

Comment:
This is actually not quite correct (although it was equally wrong before 😁 ). The message is displayed if the assert fails, which only happens when the two values are not equal to each other. 

@SgtPooki if you could push a commit that just removes the message altogether (making it `assert.strictEqual(c, inp._hash);` that would be really helpful.